modules/toolchains_llvm/0.10.2/MODULE.bazel

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 The Bazel Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module(
+    name = "toolchains_llvm",
+    version = "0.10.2",
+    compatibility_level = 0,
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.4.2")
```

Comment:
@siddharthab should skylib have `dev_dependency = True` set?

Reply:
We do make the user use `native_binary` rule from skylib, so unfortunately, this is a real dependency.